Make comments work in office for mac 2015

lexxor opened the pull request:
Hi!

Office for Mac 2015 don't show comments without 'xml:space' attribute.
Here is comments.xml extracted from xlsx file, created with Office:
```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<comments xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main">
  <authors><author>Пользователь Microsoft Office</author></authors>
  <commentList>
    <comment ref="C5" authorId="0">
      <text><r><rPr><b/><sz val="10"/><color indexed="81"/><rFont val="Calibri"/></rPr><t xml:space="preserve">test comment</t></r></text>
    </comment>
  </commentList>
</comments>
```
I can't test another office versions, but xlsx files exported from google docs also contains this attribute.

randym commented:
Thanks lexxor.

Couple of quick questions:

Where did the changes in the xsd come from? We should be using the xsd from the documentation, so maybe there is an update out there we dont know about?

Can I trouble you to add specs proving that the xml:space attribute is added?

lexxor commented:
Unfortunately, I didn't find any updates related to xml:space attribute. And I was forced to modify xsd myself to make tests green.
I added some specs for xml:space attribute.
And I removed excess attribute. Excel doesn't add this attribute for author, only for comment text. It was my fault.
